config/permissions.tmpl.yml

```diff
@@ -35,6 +35,7 @@
 - ems-type:gce_network
 - ems-type:hawkular
 - ems-type:hawkular_datawarehouse
+- ems-type:monitoring
```

Comment:
@simon3z turns out I do need it until monitoring manager has children[1]
I will remove it in #15506
[1] https://github.com/ManageIQ/manageiq/blob/f2d73802fef77276405aa5eaa2acfd2df31fb230/app/models/ext_management_system.rb#L10-L12